Mathlib/Data/List/Basic.lean

```diff
@@ -871,6 +871,11 @@ theorem get_eq_get? (l : List α) (i : Fin l.length) :
 section deprecated
 set_option linter.deprecated false -- TODO(Mario): make replacements for theorems in this section
 
+/-- nth element of a list `l` given `n < l.length`. -/
+@[deprecated get] -- 2023-01-05
```

Comment:
Want to use the `since` argument?

Reply:
I *want*to, but cannot: @deprecated accepts *either* a replacement declaration or a date. I decided the replacement was more useful, especially since `nthLe` is still used in a number of places.

Reply:
Oh huh. Can you file an issue for that?

Reply:
... turns out I was wrong :tada: the date has to come *after* the new name, though.
When going over mathlib and reformatting dates to use this syntax, this place and and should be updated as well.